src/openforms/appointments/service.py

```diff
@@ -33,12 +33,6 @@ def register_appointment(submission: Submission) -> None:
       database with the relevant context/information - the exception just signals the
       failure to the caller so that it can retry if needed.
     """
-    # TODO: implement, see PR #573
+    from .utils import book_appointment_for_submission
```

Comment:
can this import be moved to the top of the module? I'm not immediately seeing where we have a circular dependency

Reply:
`utils` imports `AppointmentRegistrationFailed` from this file so it can raise that exception, that's where the circular dependency is.  

We could add `from .service import AppointmentRegistrationFailed` in the `utils` file or move some of the code around to prevent the circular import but I'm not sure if those are better solutions.